api/swagger/swagger.yaml

```diff
@@ -9457,6 +9469,182 @@ paths:
           description: "Success"
           schema:
             $ref: "#/definitions/DocObject"
+        "403":
+          description: "Access Denied"
+          schema:
+            $ref: "#/definitions/Error"
+
+### Favorites Routes  
+  /favorites/add-to-favorites:
+    x-swagger-router-controller: favorite
+    options:
+      tags:
+        - favorite
+      summary: "Pre-flight request"
+      operationId: protectedOptions
+      description: "Options on Favorite GET Route"
+      responses:
+        "200":
+          description: "Success"
+          schema:
+            $ref: "#/definitions/FavoriteObject"
+        "403":
+          description: "Access Denied"
+          schema:
+            $ref: "#/definitions/Error"
+    post:
+      tags:
+        - Favorite
+      summary: "Create a new favorite"
+      operationId: addFavorite
+      description: "Create a new favorite object."
+      security:
+        - Bearer: []
+      x-security-scopes:
+        - project-system-admin
+      parameters:
+        - name: favorite
+          in: body
+          required: true
+          schema: 
+            $ref: "#/definitions/FavoriteObject"
+      responses:
+        "200":
+          description: "Success"
+          schema:
+            $ref: "#/definitions/FavoriteObject"
+        "403":
+          description: "Access Denied"
+          schema:
+            $ref: "#/definitions/Error"
+  /favorites/{favoriteId}:
+    x-swagger-router-controller: favorite
+    options:
+      tags:
+        - favorite
+      summary: "Pre-flight request"
+      operationId: protectedOptions
+      description: "Options on Favorite GET Route"
+      parameters:
+        - name: favoriteId
+          in: path
+          description: "ID of favorite to delete"
+          required: true
+          type: string
+      responses:
+        "200":
+          description: "Success"
+          schema:
+            $ref: "#/definitions/FavoriteObject"
+        "403":
+          description: "Access Denied"
+          schema:
+            $ref: "#/definitions/Error"
+    
+    delete:
+      tags:
+        - favorite
+      summary: "Delete a favorite"
+      operationId: removeFavorite
+      description: "Delete a favorite object."
+      security:
+        - Bearer: []
+      x-security-scopes:
+        - project-system-admin
+      parameters:
+        - name: favoriteId
+          in: path
+          description: "ID of favorite to delete"
+          required: true
+          type: string
+      responses:
+        "200":
+          description: "Success"
+          schema:
+            $ref: "#/definitions/FavoriteObject"
+        "403":
+          description: "Access Denied"
+          schema:
+            $ref: "#/definitions/Error"
+  /public/favorites/add-to-favorites:
```

Comment:
Can this URL be just /public/favourites - instead of having a verb like 'add-to-favourite'

Reply:
updated